distribution/common/src/main/resources/etc/definitions/core-attributes-validator.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Disregard my comment above.  I think this is fine here.  It could alternatively be put into the alliance app feature, but I don't know that is any more correct than what you have here.